setup.py

```diff
@@ -225,6 +226,7 @@ def check_system_cfitsio_objects(self, obj_name):
       url="https://github.com/esheldon/fitsio",
       author="Erin Scott Sheldon",
       author_email="erin.sheldon@gmail.com",
+      setup_requires=['numpy'],
```

Comment:
The goal is to rely on setuptools to install numpy if it is needed, instead of asking the user to install manually before installing fitsio. So with setup_requires setuptools will get the Numpy egg and install it temporarily, if needed, and just for the setup execution.